src/components/cards/Card.tsx

```diff
@@ -10,25 +10,32 @@
 
 import classNames from 'classnames';
 import { Spinner } from 'components/Spinner';
-import React, { ReactNode } from 'react';
+import { CSSProperties, FormEvent, ReactNode } from 'react';
 import { useTranslation } from 'react-i18next';
 import { CardContainer } from '.';
 import { Button } from '../forms';
 import { Element } from 'components/cards/Element';
+import { Dropdown } from 'components/dropdown/Dropdown';
+import { DropdownElement } from 'components/dropdown/DropdownElement';
+import { Icon } from 'components/icons/Icon';
+import { BiPlusCircle } from 'react-icons/bi';
 
 interface Props {
   children: ReactNode;
   title?: string | null;
   description?: string;
   withSaveButton?: boolean;
-  onFormSubmit?: (event: React.FormEvent<HTMLFormElement>) => any;
-  onSaveClick?: any;
+  withCreateOption?: boolean;
```

Comment:
I don't think this is good naming. 

If we gonna support multiple options for card button it shouldn't be named withCreateOption. Can we abstract this and generalize the dropdown?

Reply:
I hope that I got your idea about generalizing drop down options for save button on this component. So, I made logic something similar to `more_actions` options. Let me know your thoughts.